src/troute-nwm/src/nwm_routing/output.py

```diff
@@ -194,7 +195,11 @@ def nwm_output_generator(
 
     if test:
         flowveldepth.to_pickle(Path(test))
-    
+        import pdb; pdb.set_trace()
+        nudge = np.concatenate([r[8] for r in results])
+        usgs_positions_id = np.concatenate([r[3][0] for r in results])
+        
+        nhd_io.write_flowveldepth_netcdf(Path(qvd_ndg), flowveldepth, nudge, usgs_positions_id, t0)
```

Comment:
Let's move this into it's own logic statement rather than being nested under if test: ...